backend/src-common/src/main/java/org/eclipse/sw360/mail/MailUtil.java

```diff
@@ -113,6 +118,19 @@ public void sendMail(Set<String> recipients, String excludedRecipient, String su
         sendMail(recipients, excludedRecipient, subjectNameInPropertiesFile, textNameInPropertiesFile, notificationClass, roleName, true, textParameters);
     }
 
+    public void sendMailAsync(MimeMessage message) {
+        ExecutorService mailExecutor = Executors.newSingleThreadExecutor();
+        mailExecutor.execute(() -> {
+            try {
+                Transport.send(message);
+                log.info("Sent message successfully to user " + message.getRecipients(Message.RecipientType.TO));
+            } catch (MessagingException e) {
+                log.error("Could not sent E-Mail notification via SMTP", e);
```

Comment:
Maybe also log the reason, why the sending failed?

Reply:
but it would be good to have the e-mail and/or the STMP server address in the log

Reply:
added host